src/bci_build/package/__init__.py

```diff
@@ -402,12 +402,43 @@ def build_name(self) -> str | None:
 
     @property
     def build_version(self) -> str | None:
+        """Define the BuildVersion that is used for determining which container build
+        is newer than the other in the build service."""
+        # KIWI used to require an at least 3 component version X.Y.Z. For SLE, we set
+        # X.Y to MAJOR.MINOR and set Z to 0 for OsContainers. Derived
+        # containers inhert the base build_version X.Y and append a suffix .Z.ZZ
+
+        # It is important that the behavior for OsContainers is identical
+        # between KIWI and Dockerfile builds so that we can switch between these
+        # types.
+
+        if self.os_version.is_tumbleweed and isinstance(self, OsContainer):
+            return f"{str(datetime.datetime.now().year)}.0.0"
+        # TODO: also set it for non-kiwi type (historically we haven't done so)
+        if self.os_version.is_tumbleweed and self.build_recipe_type == BuildType.KIWI:
+            return f"{str(datetime.datetime.now().year)}.0"
```

Comment:
That's just 2025.0, without the .patch component. Intentional?

Reply:
yes, it is combined with the application version of the derived contaienr, so this leads to `2025.0.<major>` of the derived container version. compatible with how it works for sle15/sle16 so we only have one case to worry about.